packages/bits/src/lib/content/content.component.ts

```diff
@@ -10,6 +10,7 @@ import {Component, Input, ViewEncapsulation} from "@angular/core";
     templateUrl: "./content.component.html",
     styleUrls: ["./content.component.less"],
     encapsulation: ViewEncapsulation.None,
+    host: { "role": "contentinfo" },
```

Comment:
I'm not sure if contentinfo is the right role for this. The documentation for contentinfo says that it's used to identify information repeated at the end of every page of a website such as copyright info and navigation links.

![image](https://user-images.githubusercontent.com/10159693/113182519-987aff00-9218-11eb-8b67-39b6ac1e823c.png)

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/listbox_role

Reply:
will change to listitem, thanks